ci/jenkins/test-mc.sh

```diff
@@ -104,6 +104,11 @@ function clean_tmp() {
     find ${WORKDIR} -name "support-bundles*" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
 }
 
+function clean_images() {
+    docker images | tail -n +12 | grep -E 'mc-controller|antrea-ubuntu' | awk '{print $3}' | xargs -r docker rmi -f || true
```

Comment:
why tail 12 lines only? what if mc-controller and antrea-ubuntu are in the other end of the list?

Reply:
Here we actually can remove them all, I am thinking it's OK to keep top 12 latest images without checking it.

Reply:
code should be added for purpose, otherwise hard to reach consensus among a community. It doesn't have real reason to keep 12 latest images and it uses a magic number 12 that others won't get what it stands for unless they find this conversation. And if later another people removes this tail via another PR, it's technically correct as they could say there is no reason to keep 12 latest images when it's actually doing a cleanup. But these changes would really be meanlingless.

Reply:
The original purpose is to keep latest image to reduce the docker build time, I thought that there might be some shared layer can be reused, but after checking the `docker build`, it should be using build cache instead of existing image layer. I will remove it. thanks.

Reply:
Fixed